src/libertem/executor/dask.py

```diff
@@ -186,6 +186,26 @@ def get_available_workers(self):
             for worker in info['workers'].values()
         ])
 
+    def get_resource_details(self):
+        workers = self.get_available_workers()
+        details = {}
+
+        for worker in workers:
+            if worker.name.startswith("tcp"):
+                host_name = worker.host
+                resource = 'cpu'
+            else:
+                host_name, resource, _ = worker.name.split('-')
```

Comment:
What happens if the host name contains a `'-'`?

Reply:
`get_resource_details` is redundant.

Reply:
should I keep `get_resource_details` and remove the duplicate code from `ClusterDetailHandler` ?